code/modules/blob/blob.dm

```diff
@@ -152,78 +71,98 @@
 	update_icon()
 
 /obj/effect/blob/proc/expand(var/turf/T)
-	//Dont epxand over unsimulated unless its astroid trufs
-	if(istype(T, /turf/unsimulated/) && !istype(T, /turf/unsimulated/floor/asteroid/))
-		return
-
-	//Dont expand over space or holes, unless there´s a lattice
-	if((istype(T, /turf/simulated/open) || istype(T, /turf/space)) && !(locate(/obj/structure/lattice) in T))
+	if(istype(T, /turf/space) || (istype(T, /turf/simulated/open)) || (istype(T, /turf/simulated/mineral) && T.density))
```

Comment:
```suggestion
	if(istype(T, /turf/space) || (isopenturf(T)) || (istype(T, /turf/simulated/mineral) && T.density))
```

Can you add an `isspace()` macro to _macros.dm? Idk why we don't have that one.

Reply:
probably but not now or anywhere in the near future